sky/serve/infra_providers.py

```diff
@@ -82,6 +84,8 @@ def is_scale_down_no_failure(self) -> bool:
             return False
         if self.user_app_failed:
             return False
+        if self.preempted:
```

Comment:
Shall we add this attribute to `should_track_status` too?

Reply:
Thanks, done.

Reply:
It seems not resolved? I mean when `self.preempted == True`, we shouldn't send any readiness probe to that replica?